src/prognostic_equations/surface_temp.jl

```diff
@@ -0,0 +1,13 @@
+surface_temp_tendency!(Yₜ, Y, p, t, ::PrescribedSurfaceTemperature) = nothing
+
+function surface_temp_tendency!(Yₜ, Y, p, t, ::PrognosticSurfaceTemperature)
+    depth = 10 # ocean mixed layer depth [m]
+    ρ_ocean = 1020 # ocean density [kg / m³]
+    cp_ocean = 4184 # ocean heat capacity [J/(kg * K)]
```

Comment:
Should we add these as input parameters? cc @tapios

Reply:
They should stay in a parameter file in the end. But for now I think it's fine, these are not very important parameters (and not those we need to calibrate).